docs/examples/models_orm_mode_reserved_name.py

```diff
@@ -0,0 +1,27 @@
+import typing
+
+from pydantic import BaseModel, Field
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+
+class MyModel(BaseModel):
+    metadata: typing.Dict[str, str] = Field(alias="metadata_")
```

Comment:
single quotes please.

Reply:
Would you mind a PR to add flake8 linting to `docs/examples`, so you don't need to make comments like these?

Reply:
That would be great, but I think you might need to disable many error codes. The many one is that currently docs examples use one blank line everywhere, not two. I'd be happy to change that in all the examples.

Reply:
Here ya go! https://github.com/samuelcolvin/pydantic/pull/1530